torch/_inductor/config.py

```diff
@@ -1106,6 +1106,12 @@ class cpp:
     # decomposed into 7x4x2 thread blocks along MxNxK of a GEMM.
     gemm_thread_factors = os.environ.get("TORCHINDUCTOR_CPP_GEMM_THREAD_FACTORS", None)
 
+    # Set GEMM Transverse strategy: support VERTICAL, HORIZONTAL
+    # If both are enabled, the strategy will be selected based on the heuristic.
```

Comment:
Since you introduced a different strategy here, suggest to explain in the code comment what VERTICAL and HORIZONTAL traversal mean. This allows users/developers to know the meaning of the configurations.

Also, I guess you misspell the word - "traversal" instead of "transverse"?